qiskit_bot/release_process.py

```diff
@@ -234,14 +235,16 @@ def _generate_changelog(repo, log_string, categories, show_missing=False):
     return changelog
 
 
-def create_github_release(repo, log_string, version_number, categories):
+def create_github_release(repo, log_string, version_number, categories,
+                          prerelease=False):
     changelog = _generate_changelog(repo, log_string, categories)
     release_name = repo.name + ' ' + version_number
-    repo.gh_repo.create_git_release(version_number, release_name, changelog)
+    repo.gh_repo.create_git_release(version_number, release_name, changelog,
+                                    prerelease=prerelease)
 
 
-def _get_log_string(version_number_pieces):
-    version_number = '.'.join(version_number_pieces)
+def _get_log_string(version_number_pieces, version_string):
+    version_number = version_string
```

Comment:
I'm not about to hold up the PR over it, but since the caller of this function has the parsed `packaging.version` object, we could just pass that as a singleton object into this helper function and also get rid of all the magic numbers in the indexing below.

Reply:
Yeah that's a good idea I'll do this, it would be a lot cleaner

Reply:
Done in: https://github.com/Qiskit/qiskit-bot/pull/24/commits/255d475c38d60881eae87c2a9b813a30480c1fbc